test/e2e/shoot/logging/event_logging.go

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logging
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/gardener/pkg/utils/retry"
+	"github.com/gardener/gardener/test/e2e"
+	"github.com/gardener/gardener/test/framework"
+
+	"github.com/go-logr/logr"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	eventLogger            = "event-logger"
+	eventMessageForSeed    = "Event from Event Logger integration test related to the Seed"
+	eventMessageForShoot   = "Event from Event Logger integration test related to the Shoot"
+	seedEventName          = "seed-event"
+	shootEventName         = "shoot-event"
+	eventLoggerName        = "event-logger"
+	kubeSystemNamespace    = "kube-system"
+	operatorID             = "operator"
+	lokiName               = "loki"
+	gardenNamesapce        = "garden"
+	fluentBitDaemonSetName = "fluent-bit"
+)
+
+var _ = Describe("Seed logging testing", Label("Shoot", "logging"), func() {
+
+	var (
+		shootPurpose   = gardencorev1beta1.ShootPurposeDevelopment
+		shootNamespace string
+		lokiLabels     = map[string]string{
+			"app":  "loki",
+			"role": "logging",
+		}
+	)
+
+	f := defaultShootCreationFramework()
```

Comment:
Generally, I don't think it's necessary to add another test case and create another shoot just for verifying the `event-logger`.
Instead, I would rather add an additional verification step to an existing e2e test case, e.g. after https://github.com/gardener/gardener/blob/d340d78b17261862a5e1cc8b6648e9b58ab1314d/test/e2e/shoot/create_update_delete.go#L57-L63

You could move the verification logic into `internal/logging` and expose it as some kind of "verifier" similar to https://github.com/gardener/gardener/blob/d340d78b17261862a5e1cc8b6648e9b58ab1314d/test/e2e/shoot/internal/rotation/observability.go